test/setup_bundler_test.rb

```diff
@@ -67,12 +85,17 @@ def run_script
     $LOADED_FEATURES.delete(path)
   end
 
-  def bundle_install_command
+  def bundle_install_command(bundle_gemfile = nil, update = true)
```

Comment:
```suggestion
  def bundle_install_command(bundle_gemfile = nil, update: true)
```
To make the caller slightly more readable.